content/README.md

```diff
@@ -0,0 +1,16 @@
+# source{d} blog
+
+Here is the content published at the [source{d} blog](https://blog.sourced.tech).
+
+
+# Contributing
+
+If you want to contribute to this project, or to write or edit a blog post, you will find more info in the [contribution guidelines](../CONTRIBUTING.md).
```

Comment:
If you want to contribute to this project, write a new blog post, or edit an existing one, you can follow the [contribution guidelines](../CONTRIBUTING.md).